### DISCUSSION TITLE
希望给机器人戳一戳增加

### DISCUSSION THREAD

**Discussion by Pessqiq1** [FloatTech/ZeroBot-Plugin#623] [Ideas]
想给戳一戳加一个
（文本）+（戳的人的用户名）+文本
不知道该怎么实现

**Comment by fumiama** [comment]
用ctx.CardOrNickName()，我记得是这个名字。

**Reply by Pessqiq1** [reply]
可以给个详细命令行吗 我这里编译一直报错

**Reply by fumiama** [reply]
你难道是用记事本写的代码吗？如果是的话，推荐用vscode，有详细的代码建议。至于具体函数名，我记不清名字了，有谁知道的话给他说一下吧。

**Comment by lianhong2758** [comment]
https://github.com/lianhong2758/ZeroBot-Plugin/blob/xxxppp/plugin/chat/chat.go   戳一戳禁言,很好玩(躲),你要的用户名函数ctx.CardOrNickName(uid)
